tools/MarkdownConverter/Converter/MarkdownSourceConverter.cs

```diff
@@ -951,7 +944,9 @@ IEnumerable<OpenXmlElement> Literal2Elements(string literal, bool isNested, bool
         }
     }
 
-    IEnumerable<OpenXmlCompositeElement> GenerateCustomBlockElements(string customBlockId, MarkdownParagraph.InlineBlock block) => customBlockId switch
+    // Note for future PR: "function_members", "format_strings_1", and "format_strings_2" are the
+    // only special cases still used. The others can be safely removed in a future PR.
+    IEnumerable<OpenXmlCompositeElement> GenerateCustomBlockElements(string customBlockId, MarkdownParagraph.InlineHtmlBlock block) => customBlockId switch
```

Comment:
@jskeet Let me know if you want me to remove the (now) unused custom blocks in this PR.

Reply:
Looking into it now - I need to understand the comment a bit more first.